src/main/scala/org/ergoplatform/settings/ErgoValidationSettings.scala

```diff
@@ -16,12 +20,17 @@ import scala.util.Try
   *
   * @param rules - map from rule id to it's current status
   */
-case class ErgoValidationSettings(rules: Map[Short, RuleStatus]) extends ValidationSettings with BytesSerializable {
```

Comment:
I propose to:
- replace `sigmaRules: Map[Short, SRuleStatus]` to `sigmaSettings: SigmaValidationSettings`
- implement correct `ErgoValidationSettingsSerializer` that serialize `ErgoValidationSettings` including `SigmaValidationSettings`
- Old `parseExtension` code will work correctly and will require no more changes